web-share/share-null-manual.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share null and undefined values</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        // Expect null to be converted into the string 'null'. (This isn't good
+        // practice, but it's how Web IDL specifies conversion.) On the other
```

Comment:
This is good point actually. Allowing null to equal "this document" instead of empty string might be nice... or is there precedence in the platform for interpreting the empty string as this URL?